nixos/lib/test-driver/nixos_test_driver/doc/index.rst

```diff
@@ -0,0 +1,20 @@
+.. NixOS test driver documentation master file, created by
+   sphinx-quickstart on Sat Nov 16 12:56:43 2019.
+   You can adapt this file completely to your liking, but it should at least
+   contain the root `toctree` directive.
+
+Welcome to NixOS test driver's documentation!
```

Comment:
is this the title that would be printed on top of the whole document? If so - how about simply making this `NixOS test driver documentation`